hadoop-tools/hadoop-azure/src/test/java/org/apache/hadoop/fs/azurebfs/ITestAbfsStatistics.java

```diff
@@ -133,59 +159,48 @@ public void testDeleteStatistics() throws IOException {
         + "in Abfs");
     AzureBlobFileSystem fs = getFileSystem();
     /*
-     This directory path needs to be root for triggering the
-     directories_deleted counter.
+    This directory path needs to be root for triggering the
+    directories_deleted counter.
      */
     Path createDirectoryPath = path("/");
     Path createFilePath = path(getMethodName());
 
-    Map<String, Long> metricMap = fs.getInstrumentation().toMap();
-    //Test for initial statistic values before deleting any directory or files
```

Comment:
Initial stats should be asserted to have values zero. I asked to move in a separate method and reuse in all tests.

Reply:
This is outdated code, I have moved all initial values into a single method.